<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>804</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="chartWidget" native="true">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>380</y>
      <width>921</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>119</x>
      <y>0</y>
      <width>276</width>
      <height>374</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Input file:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="linePath"/>
       </item>
       <item>
        <widget class="QPushButton" name="btnBrowse">
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Algorithm:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboAlg"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLineEdit" name="lineSim"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Simulation time:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRun">
       <property name="text">
        <string>Run Simulation</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="output"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
